.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: release
+
+on:
+  create:
```

Comment:
Should this be `push:` instead of `create:`

Reply:
both works for tags in this case, I can change for push, if you prefer

Reply:
updated to push